docs/pages/more/expo-cli.mdx

```diff
@@ -68,19 +68,26 @@ The UI that shows up in the process is referred to as the **Terminal UI**. It co
+| <kbd>I</kbd>                    | Open the project in an iOS simulator.                                                                                                                                                    |
 | <kbd>Shift</kbd> + <kbd>I</kbd> | Select an iOS Simulator to open.                                                                                                                                                         |
 | <kbd>W</kbd>                    | Open the project in a web browser. This may require webpack to be installed in your project.                                                                                             |
 | <kbd>R</kbd>                    | Reload the app on any connected device.                                                                                                                                                  |
+| <kbd>S</kbd>                    | Switch the launch target between Expo Go and development builds.                                                                                                                         |
 | <kbd>M</kbd>                    | Open the dev menu on any connected native device (web not supported).                                                                                                                    |
 | <kbd>Shift</kbd> + <kbd>M</kbd> | Choose more commands to trigger on connected devices.<br/>This includes toggling the performance monitor, opening the element inspector, reloading the device, and opening the dev menu. |
 | <kbd>J</kbd>                    | Open Chrome Dev Tools for any connected device that is using Hermes as the JavaScript engine. [Learn more](/guides/using-hermes#javascript-inspector-for-hermes).                        |
 | <kbd>O</kbd>                    | Open project code in your editor. This can be configured with the `EXPO_EDITOR` and `EDITOR` [environment variables](#environment-variables).                                            |
 | <kbd>E</kbd>                    | Show the development server URL as a QR code in the terminal.                                                                                                                            |
 | <kbd>?</kbd>                    | Show all Terminal UI commands.                                                                                                                                                           |
 
+### Launch target
+
+The `npx expo start` command will automatically launch the app in Expo Go unless `expo-dev-client` in installed in the project, in which case it will open in the custom app. You can force the launch target by passing the `--dev-client` and `--go` flags. For example, running `npx expo start --go` will always launch the app in Expo Go, and `npx expo start --dev-client` will always attempt to launch the app in a development build.
```

Comment:
the case this doesn't handle is a native app without expo-dev-client. `npx expo start` in a react-native app w/ ios and android directories probably shouldn't default to expo go

Reply:
it's fine for now as this is the existing behavior and we don't break anything by merging new functionality for a different case. If users are doing development in an unintended way, then they may need to pass an additional flag (already the default), or switch the mode by pressing `s`.